lib/tool_belt/cherry_picker.rb

```diff
@@ -92,7 +93,11 @@ def write_cherry_pick_log(picks, release)
     end
 
     def log_entry(pick)
-      "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+      if pick['bugzilla_id']
+        "#{pick['bugzilla_id']} - #{pick['id']} : [#{pick['revision']}] #{pick['subject']}"
+      else
+        "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+      end
```

Comment:
If I look at https://github.com/ehelms/tool_belt/pull/19/files#diff-008c904eac6e32bcbddea7437dbb73d8R51 it pulls the bugzilla_id from the issue data, which would imply that some objects when they reach this point in the code may have that populated and some may not (aka issues without bugzilla's attached). Which would result in varried output format.